src/components/Form/DeleteDocumentForm.jsx

```diff
@@ -50,30 +55,29 @@ const DeleteDocumentForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
-  /* eslint-disable jsx-a11y/label-has-associated-control */
   return (
     <FormSection
       title="Delete Document"
       state={state}
       statusType="Deletion status"
       defaultMessage="To be deleted..."
     >
-      <form onSubmit={handleDeleteDocument} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="delete-doctype">Select document type to delete: </label>
-          <DocumentSelection htmlId="delete-doctype" />{' '}
-          <button disabled={state.processing} type="submit">
-            Delete Document
-          </button>
-        </div>
-      </form>
+      <Box display="flex" justifyContent="center" sx={{ minWidth: 120 }}>
+        <form onSubmit={handleDeleteDocument}>
+          <InputLabel id="delete-doctype">
+            <i>Select Document Type to Delete:</i>
+          </InputLabel>
+          <FormControl variant="standard" fullWidth>
+            <DocumentSelection htmlId="search-doctype" />
```

Comment:
**line 71** - `htmlID` should be "delete-doctype" to match the `id` of the label to link the two together.

Reply:
Good catch!